### ISSUE TITLE
Add a way to prepare the repo easily before starting to contribute

### ISSUE THREAD

**Issue by aurelien-reeves** [cucumber/html-formatter#3]
As the java and ruby implementation rely on the javascript one, some preparation need to be done before starting working on the html-formatter:
- building the javascript package
- copy a bunch of files from the javascript folder into the jvm and ruby ones

(take a look on the CI workflows for more info on what need to be done)

It could be nice to have a single script / makefile / whatever to make it easy for contributors to get started
